modules/cluster-logging-deploy-clo-cli.adoc

```diff
@@ -0,0 +1,266 @@
+        node-role.kubernetes.io/worker: ""
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  collection:
+    logs:
+      type: "fluentd"
+      fluentd: {}
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+endif::[]
+
+ifdef::openshift-enterprise,openshift-origin[]
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance" <1>
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"  <2>
+  logStore:
+    type: "elasticsearch"  <3>
+    elasticsearch:
+      nodeCount: 3 <4>
+      storage:
+        storageClassName: gp2 <5>
+        size: 200G
+      redundancyPolicy: "SingleRedundancy"
+  visualization:
+    type: "kibana"  <6>
+    kibana:
+      replicas: 1
+  curation:
+    type: "curator"  <7>
+    curator:
+      schedule: "30 3 * * *"
+  collection:
+    logs:
+      type: "fluentd"  <8>
+      fluentd: {}
+----
+<1> The name must be `instance`.
+<2> The cluster logging management state. In most cases, if you change the default cluster logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until the cluster logging is placed back into a managed state. For more information, see *Changing cluster logging management state*.
+<3> Settings for configuring Elasticsearch. Using the CR, you can configure shard replication policy and persistent storage. For more information, see *Configuring Elasticsearch*.
+<4> Specify the number of Elasticsearch nodes. See the note that follows this list.
+<5> Specify that each Elasticsearch node in the cluster is bound to a Persistent Volume Claim.
+<6> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana nodes. For more information, see *Configuring Kibana*.
+<7> Settings for configuring Curator. Using the CR, you can set the Curator schedule. For more information, see *Configuring Curator*.
+<8> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring Fluentd*.
+endif::[]
++
+[NOTE]
++
+====
+The maximum number of Elasticsearch master nodes is three. If you specify a `nodeCount` greater than `3`, {product-title} creates three Elasticsearch nodes that are Master-eligible nodes, with the master, client, and data roles. The additional Elasticsearch nodes are created as Data-only nodes, using client and data roles. Master nodes perform cluster-wide actions such as creating or deleting an index, shard allocation, and tracking nodes. Data nodes hold the shards and perform data-related operations such as CRUD, search, and aggregations. Data-related operations are I/O-, memory-, and CPU-intensive. It is important to monitor these resources and to add more Data nodes if the current nodes are overloaded.
```

Comment:
Maybe leaving a note here of what happens if someone request `nodeCount < 3`. What will happen with regard to the `redundancyPolicy`?